### PR TITLE
yiisoft: Added CVE-2018-7269 and related CVE-2018-8073, CVE-2018-8074

### PR DESCRIPTION
**Pull request by SilverFire**

### CODE REVIEW

File: yiisoft/yii2-elasticsearch/CVE-2018-8074.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     Possibility of manipulated condition when unfiltered input is passed to `yii\elasticsearch\ActiveRecord::findOne()` and `::findAll()`
+link:      http://www.yiiframework.com/news/168/releasing-yii-2-0-15-and-database-extensions-with-security-fixes/
+cve:       CVE-2018-8072
```
Comment (cebe) [inline yiisoft/yii2-elasticsearch/CVE-2018-8074.yml]: should be CVE-2018-8074.

File: yiisoft/yii2/CVE-2018-7269.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title:     Potential SQL injection in methods `yii\db\ActiveRecord::findOne()` and `::findAll()`
+link:      http://www.yiiframework.com/news/168/releasing-yii-2-0-15-and-database-extensions-with-security-fixes/
+cve:       CVE-2018-7269
+branches:
+    2.0.x:
+        time:     2018-03-20 14:00:00
+        versions: ['<2.0.12.1', '2.0.13', '2.0.13.1', '~2.0.14.0']
+reference: composer://yiisoft/yii2
```
Comment (cebe) [inline yiisoft/yii2/CVE-2018-7269.yaml]: same file needs to be added to yii2-dev repo.

Comment (SilverFire) [reply yiisoft/yii2/CVE-2018-7269.yaml]: +

File: yiisoft/yii2-dev/CVE-2018-7269.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+title:     Potential SQL injection in methods `yii\db\ActiveRecord::findOne()` and `::findAll()`
+link:      http://www.yiiframework.com/news/168/releasing-yii-2-0-15-and-database-extensions-with-security-fixes/
+cve:       CVE-2018-7269
+branches:
+    2.0.x:
+        time:     2018-03-20 14:00:00
+        versions: ['<2.0.12.1', '2.0.13', '2.0.13.1', '~2.0.14.0']
```
Comment (cebe) [inline yiisoft/yii2-dev/CVE-2018-7269.yaml]: looks like the validator requires `=` for matching equal versions. `~` seems to be not allowed

**Comment by cebe** [comment]
fixed branches version constraints in https://github.com/FriendsOfPHP/security-advisories/pull/277

**Pull request merged**